lib/merch_calendar.rb

```diff
@@ -1,12 +1,12 @@
+require "date"
 
 #
 module MerchCalendar
-
```

Comment:
The deprecation date applies to the methods on the `MerchCalendar` singleton object.  So like...

`MerchCalendar.start_of_week` is deprecated in favor of `MerchCalendar::RetailCalendar.new.start_of_week`

so I feel like it's appropriate here.

Though I think we may be disconnected in what I was trying to achieve.  LMK if that still doesn't make sense after the additional explanation.